docs/modules/ROOT/pages/explanations/webhooks.adoc

```diff
@@ -0,0 +1,30 @@
+= Webhooks
+
+== APPUiO Cloud quota webhooks
+APPUiO Cloud provides a quota system, which AppCat has to respect and integrate into.
+It accomplishes that, by providing nice error messages, if any of the given quotas are breached.
+
+There are multiple quotas that need to be respected by this system:
+* the amount of namespaces an organization can have
+* the amount of resources that can be used within a namespace
+
+The amount of namespaces is straight forward by checking if there's a custom value set or just checking against the default of 25.
```

Comment:
```suggestion
The number of namespaces is straight forward by checking if there's a custom value set or just checking against the default of 25.
```

Reply:
I agree with @wejdross `amount` is IMHO the right word here.